### PR TITLE
Qt cleanup

### PR DESCRIPTION
**Pull request by yuriks**

### CODE REVIEW

**Comment by mailwl** [comment]
btw, Qt is upated to 5.8...

File: src/citra_qt/main.cpp [hunk 1]
```diff
@@ -290,12 +290,12 @@ bool GMainWindow::LoadROM(const std::string& filename) {
...
     Core::System& system{Core::System::GetInstance()};
 
-    const Core::System::ResultStatus result{system.Load(render_window, filename)};
+    const Core::System::ResultStatus result{system.Load(render_window, filename.toStdString())};
 
     if (result != Core::System::ResultStatus::Success) {
         switch (result) {
         case Core::System::ResultStatus::ErrorGetLoader:
-            LOG_CRITICAL(Frontend, "Failed to obtain loader for %s!", filename.c_str());
+            LOG_CRITICAL(Frontend, "Failed to obtain loader for %s!", filename.toUtf8().data());
```
Comment (wwylele) [inline src/citra_qt/main.cpp]: I had to look for documentation to confirm `data()`is null-terminated... can be just `filename.toStdString().c_str()` which is clearer and does the same thing.

**Comment by paulconrad12345** [comment]
The latest bleeding build won't open :(? It crash

**Comment by jfmherokiller** [comment]
sorry for not making IsSingleFileDropEvent static in the first place

File: src/citra_qt/main.cpp (hunk 1 above)
Comment (yuriks) [inline src/citra_qt/main.cpp]: Fair enough, I'll change it.

**Pull request merged**